tasks/gulp/backstop-config.js

```diff
@@ -83,7 +86,7 @@ module.exports = ({ host, port, components }) => ({
     args: ['--no-sandbox'],
   },
   // if running locally, and either stage hangs, try reducing these limits
-  asyncCaptureLimit: 5,
+  asyncCaptureLimit: 3,
```

Comment:
this is reduced because the number of scenarios has doubled in the presence of rebrand for a bit - and having it at 5 was causing some checks to fail for me